receiver/jaegerreceiver/metrics.go

```diff
@@ -0,0 +1,55 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package jaegerreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver"
+
+import (
+	"github.com/jaegertracing/jaeger/pkg/metrics"
+)
+
+const (
+	DroppedPacketsStat   = "thrift.udp.server.packets.dropped"
```

Comment:
To clarify, these are the names that thrift server is using to report the metrics, but you're re-mapping them to counters generated from metadata? I would suggest adding a comment to that effect.

Any reason not to keep the metric names identical to Jaeger, for better compatibility?

Reply:
Hello 

> To clarify, these are the names that thrift server is using to report the metrics, but you're re-mapping them to counters generated from metadata?

Yes, that's right.

> Any reason not to keep the metric names identical to Jaeger, for better compatibility?

Unfortunately, it is currently not possible to retrieve counter names from the Jaeger SDK as t[hese names are specified using private/non-exportable field tags](https://github.com/jaegertracing/jaeger/blob/3bc0d35849ac78fe888b33f4a353bc6ee78efa78/cmd/agent/app/servers/tbuffered_server.go#L52).

> I would suggest adding a comment to that effect.

I can add a corresponding comment to the counters names definition.

Reply:
> Unfortunately, it is currently not possible to retrieve counter names from the Jaeger SDK

You can copy the names but keep them the same. Right now they are changed to completely different metrics.